src/subscription/data/service.js

```diff
@@ -0,0 +1,50 @@
+import { ensureConfig, getConfig } from '@edx/frontend-platform';
+import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
+
+import handleRequestError from '../../payment/data/handleRequestError';
+import { transformResults } from '../../payment/data/utils';
+
+ensureConfig([
+  'ECOMMERCE_BASE_URL',
+  'LMS_BASE_URL',
+], 'payment API service');
+
+function handleDetailsApiError(requestError) {
```

Comment:
Is a different but repeated handle function here for handling errors necessary or could we use the same as regular payment flow?

Reply:
I think we should keep it, we are not sure what error structure would be from the new Subs BE service. it will be easier to modify it according to our needs in this way.

Reply:
That's a fair point, we don't know what the BE error structure will be like. In this case, can you either add a TODO or leave the function empty? Because I'd rather not have code in there that we don't know if it works.

Reply:
added TODO